code/game/objects/structures/tables_racks.dm

```diff
@@ -136,6 +136,7 @@
 		if(istype(src, /obj/structure/table/optable))
 			var/obj/structure/table/optable/OT = src
 			G.affecting.resting = 1
+			G.affecting.update_canmove()

```

Comment:
So there's no more delay between the moment you put the patient on the table and the moment he decides to lie down.